cmake/dependencies/imath.cmake

```diff
@@ -29,6 +29,7 @@ SET(_install_dir
 SET(_include_dir
     ${_install_dir}/include/Imath
 )
+SET(RV_DEPS_IMATH_ROOT_DIR ${_install_dirs})
```

Comment:
Wrong name ❗ There's an extra 's' at the end of the name.

Reply:
Good catch ! Sorry about that